communication/slack-config/sig-node/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - sig-node-leads
+approvers:
+  - sig-node-leads
```

Comment:
Do we want this to be `sig-node-reviewers` and `sig-node-approvers` instead of `sig-node-leads`? As `leads` only two people will have the ability to approve PRs here.

Reply:
Up to you. What do you prefer?

Reply:
What do the other sigs do here?

Reply:
Looks like all of the others use `leads` so we should probably do the same.